src/main/java/com/exasol/ciisolation/aws/cleanup/AwsAccountCleanerBuildSpec.java

```diff
@@ -32,6 +32,7 @@
         return new String[] { getWriteToFileCommand(getAwsNukeConfig(), AWS_NUKE_CONFIG), //
                 "cat aws-nuke-config.yml", //
                 "wget -q https://github.com/rebuy-de/aws-nuke/releases/download/v2.15.0/aws-nuke-v2.15.0-linux-amd64.tar.gz -O aws-nuke.tar.gz", //
+                "echo \"7d8b50fec337a546e22d1d4d0a2fd3363f140f38  aws-nuke.tar.gz\" | shasum -c",
```

Comment:
This would have to change with each update. Unfortunately, the aws-nuke artifacts do not have corresponding checksum in the assets list. Then you could download both of them.

Reply:
Yes, that is good point. 

Still when you update the version you get the shasum from downloaded artifact. More secure variant would be gpg signature checks using developers keys in addition to checksum checks. An example using gpg keys [here](https://github.com/morazow/docker-nginx/blob/main/Dockerfile#L79-L83).

Reply:
yea. That would be great. But at the moment, I think it's ok like that.